script/c511009531.lua

```diff
@@ -52,12 +52,12 @@ function c511009531.spop(e,tp,eg,ep,ev,re,r,rp)
 	end
 end
 function c511009531.vfilter(c)
-	return c:IsFaceup() and c:IsSetCard(0xaf)
+	return c:IsFaceup() and c:IsSetCard(0x10af)

```

Comment:
why was this changed to D/D/D ?
http://yugioh.wikia.com/wiki/D/D/D_Gust_High_King_Executive_Alexander